common/utils.py

```diff
@@ -20,6 +20,12 @@
     "Language", [(l[0].upper(), l[0]) for l in settings.LANGUAGES]
 )
 
+LINKED_EVENT_DATE_FORMAT = "%Y-%m-%d"
+
+
+def format_linked_event_date(datetime):
+    return datetime.strftime(LINKED_EVENT_DATE_FORMAT)
```

Comment:
Nitpickers would say you shouldn't have a variable named `datetime`which masks the datetime module.